sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/util/LoggingUtil.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.util;
+
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.core.util.logging.ClientLogger;
+
+/**
+ * Utilities to handle logging for credentials.
+ */
+public final class LoggingUtil {
+    /**
+     * Log a success message for a getToken() call.
+     * @param credential the credential class
+     * @param logger the logger to output the log message
+     * @param context the context of the getToken() request
+     */
+    public static void getTokenSuccess(Class<?> credential, ClientLogger logger, TokenRequestContext context) {
+        logger.info("Azure Identity => getToken() result for {}: SUCCESS", credential.getSimpleName());
```

Comment:
The `logger` should be created with the classname and the log statement will have with the name of the class. Do we need to print the classname again? Can drop the `Class<T> credential` arg if it's not required.

Reply:
Make sense